src/main/java/com/mattmalec/pterodactyl4j/client/entities/Database.java

```diff
@@ -0,0 +1,19 @@
+package com.mattmalec.pterodactyl4j.client.entities;
+
+import com.mattmalec.pterodactyl4j.DataType;
+import com.mattmalec.pterodactyl4j.PteroAction;
+import com.mattmalec.pterodactyl4j.client.managers.DatabaseAction;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.UUID;
+
+public interface Database {
```

Comment:
Since there is also an application implementation of databases in the Pterodactyl API, this should be named ClientDatabase. This should also extend a generic Database interface that has elements that persist in both the client and application implementations.

The application implementation includes an id, created date, and modified date, so that implementation should extend ISnowflake.

```suggestion
public interface ClientDatabase extends Database {
```

Reply:
Why is this empty?